README.md

```diff
@@ -2,20 +2,37 @@
 
 Simple, foolproof standard for signing arbitrary data.
 
-*   Why not [JOSE/JWS/JWT](https://jwt.io)? JSON-specific, too complicated, too
-    easy to mess up.
-*   Why not [PASETO](https://paseto.io)? JSON-specific, too opinionated.
+## Features
+
+*   Supports arbitrary message encodings, not just JSON.
+*   Authenticates the message *and* the type to avoid confusion attacks.
+*   Avoids canonicalization to reduce attack surface.
+*   Allows any desired crypto primitives or libraries.
+
+See [Background](background.md) for more information, including design
+considerations and rationale.
 
 ## What is it?
 
-*   [Signature protocol](specification.md)
-*   [Data structure](specification.md) for storing the message and signatures
+Specifications for:
+
+*   [Protocol](protocol.md) (*required*)
+*   [Data structure](envelope.md), a.k.a. "Envelope" (*recommended*)
```

Comment:
💯  for "Envelope" instead of "Wrapper". The latter always had a different connotation for me.